src/repr/src/row/encode.rs

```diff
@@ -1332,6 +1400,21 @@ impl ColumnDecoder<Row> for RowColumnarDecoder {
         nullability.is_null(idx)
     }
 
+    fn goodbytes(&self) -> usize {
+        let decoders_size: usize = self
+            .decoders
+            .iter()
+            .map(|(_name, _null_count, decoder)| decoder.goodbytes())
+            .sum();
+
+        decoders_size
+            + self
+                .nullability
```

Comment:
Similarly, the nullability array isn't counted in `goodbytes`.

Reply:
Done!

Reply:
I think this is still counting nulls, which we don't normally do!